graphrag_sdk/steps/extract_data_step.py

```
@@ -195,7 +200,7 @@ def _process_source(
 
         except Exception as e:
             logger.exception(e)
-            raise e
+            raise ValueError(f"Error processing source: {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)